### PR TITLE
Configure project for deployment on Heroku

### PR DESCRIPTION
**Pull request by mikescor**

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+web: gunicorn bot.wsgi --log-file -
+beat: celery -A bot worker -B
+release: python manage.py migrate
```
Comment (andriisoldatenko) [inline Procfile]: new line

File: bot/settings.py [hunk 2]
```diff
@@ -37,12 +40,21 @@
...
+]
+
+INSTALLED_APPS += [
     'twitterbot.apps.TwitterbotConfig',
-    'django_celery_results'
 ]
 
 MIDDLEWARE = [
     'django.middleware.security.SecurityMiddleware',
+    'whitenoise.middleware.WhiteNoiseMiddleware',
```
Comment (andriisoldatenko) [inline bot/settings.py]: why do we need this?

File: twitterbot/models.py [hunk 3]
```diff
@@ -18,8 +18,10 @@ def __str__(self):
 
 
 class Followers(models.Model):
-    user_id1 = models.ForeignKey(TwitterUser, on_delete=models.CASCADE, related_name='publisher')
-    user_id2 = models.ForeignKey(TwitterUser, on_delete=models.CASCADE, related_name='subscriber')
+    user_id1 = models.ForeignKey(TwitterUser, on_delete=models.CASCADE,
```
Comment (andriisoldatenko) [inline twitterbot/models.py]: strange fix

File: utils/twitterbot.py [hunk 4]
```diff
@@ -10,7 +10,8 @@
 
 
 def main(limit=1000):
-    list_users = TargetTwitterAccounts.objects.values_list('twitter_url', flat=True)
+    list_users = TargetTwitterAccounts.objects.values_list('twitter_url',
```
Comment (andriisoldatenko) [inline utils/twitterbot.py]: same as previous

File: twitterbot/models.py (hunk 3 above)
Comment (mikescor) [reply twitterbot/models.py]: The line before fix was over 79 characters.

File: utils/twitterbot.py (hunk 4 above)
Comment (mikescor) [reply utils/twitterbot.py]: the same (the line before fix was over 79 characters)

File: twitterbot/management/commands/getfollowers.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        auth = tweepy.OAuthHandler(CONSUMER_KEY, CONSUMER_SECRET)
+        auth.set_access_token(ACCESS_TOKEN, ACCESS_TOKEN_SECRET)
+        api = tweepy.API(auth)
+
+        for user in options['username']:
+
+            self.stdout.write('Loading followers of {}'.format(user))
+
+            try:
+                followers_list = api.get_user(user).followers()
```
Comment (klochenok) [inline twitterbot/management/commands/getfollowers.py]: you also need to get friends of user and save them to DB
`api.get_user(user).friends()`

**Pull request merged**